### PR TITLE
Example uses const on a non-const fn 2

### PR DESCRIPTION
**Pull request by gilescope**
Continuation of 
https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/1055
but using the source branch!

### CODE REVIEW

**Comment by gilescope** [comment]
@NukeManDan better?

**Review by NukeManDan — APPROVED** [review]
Thanks for finding and resolving the issue 👍🏼

File: docs/tutorials/add-contracts-pallet.md [hunk 1]
```diff
@@ -219,11 +219,11 @@ impl pallet_timestamp::Config for Runtime {
 
 /*** Add This Block ***/
 parameter_types! {
-	pub const TombstoneDeposit: Balance = deposit(
+	pub TombstoneDeposit: Balance = deposit(
```
Comment (NukeManDan) [inline docs/tutorials/add-contracts-pallet.md:222]:
```suggestion
	pub TombstoneDeposit: Balance = deposit( 
```

**Review by NukeManDan — APPROVED** [review]
Trigger ci

**Comment by gilescope** [comment]
@NukeManDan had to fix a broken link - please glance over the change. (Sentry nodes were deprecated)

**Comment by gilescope** [comment]
All green but I don't have a merge button. @NukeManDan can you please do the honors?

**Pull request merged**